jib-core/src/main/java/com/google/cloud/tools/jib/tar/TarStreamBuilder.java

```diff
@@ -77,7 +78,7 @@ public void addByteEntry(byte[] contents, String name, Instant modificationTime)
     TarArchiveEntry entry = new TarArchiveEntry(name);
     entry.setSize(contents.length);
     entry.setModTime(modificationTime.toEpochMilli());
-    archiveMap.put(entry, Blobs.from(outputStream -> outputStream.write(contents)));
+    archiveMap.put(entry, Blobs.from(outputStream -> outputStream.write(contents), false));
```

Comment:
I think true. (`contents` isn't totally immutable as it's a reference to an array, but this is implied and assumed in the existing code.)
```suggestion
    archiveMap.put(entry, Blobs.from(outputStream -> outputStream.write(contents), true));
```